Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/command.py

```diff
@@ -36,6 +37,31 @@ class Command(object):
     aliases = []
     help = None
 
+    @classmethod
+    def commit_message_environment_path(cls, repository):
+        if not repository:
+            return None
+        return os.path.join(repository.root_path, '.git', 'commit-message-environment.json')
```

Comment:
This won't work in a worktree where `.git` is a file; we should probably get `$GIT_DIR` here instead. (We have `repository.common_directory` which gives `$GIT_COMMON_DIR`, but we don't seem to have a `repository.git_dir` or similar.)

Reply:
Seems like we should just use `common_directory`? I struggle to think of a circumstance where that would be wrong.

Reply:
With worktrees you can have multiple commits in progress at once, so you could have two partial commits (e.g., waiting on $EDITOR to exit), and therefore we need to have `commit-message-environment.json` in per-worktree directories.

Reply:
Ah, so we do need to use `GIT_DIR` where as identifiers (which are the other primary consumer here) need to use `GIT_COMMON_DIR`